File: IDEA/Scala/src/main/scala/scala01/Person.scala
package scala01

/**
  * @title: Person
  * @projectName Scala
  * @description: TODO
  * @author ½ðÓã
  * @date 2021/12/610:47
  */
class Person {
  private var name : String = "chen"
  var id:Int = 30
  var age:Int = 20
  private[scala01] var sex:String = "ÄÐ"

  def printinfo():Unit = {
    println(s"Master:$name , $age , $id , $sex")
  }
}

File: IDEA/Spring5/src/com/chen/spring5/domain/Person.java
package com.chen.spring5.domain;

/**
 * @author Н№гу
 * @title: Person
 * @projectName Spring5
 * @description: TODO
 * @date 2021/11/228:43
 */
public class Person {
    private String aname;

    public void setAname(String aname) {
        this.aname = aname;
    }

    @Override
    public String toString() {
        return aname;
    }
}
